src/analysis/screening.jl

```diff
@@ -143,3 +143,80 @@ A shortcut for [`screen`](@ref) that only works with model variants.
 """
 screen_variants(model, variants, analysis; workers = [myid()]) =
     screen(model; variants = variants, analysis = analysis, workers = workers)
+
+"""
+    screen_optimize_objective(_, optmodel)::Union{Float64,Nothing}
+
+A variant of [`optimize_objective`](@ref) directly usable in
+[`screen_optmodel_modifications`](@ref).
+"""
+screen_optimize_objective(_, optmodel)::Union{Float64,Nothing} =
+    optimize_objective(optmodel)
+
+"""
+    screen_optmodel_modifications(
+        model::MetabolicModel,
+        optimizer;
+        modifications::Array{V,N},
+        analysis = screen_optimize_objective,
+        workers = [myid()],
+    ) where {V<:AbstractVector,N}
+
+Screen multiple modifications of the same optimization model.
+
+This function is potentially more efficient than [`screen`](@ref) because it
+avoids making variants of the model structure and remaking of the optimization
+model, but on the other hand can not express the scale of modifications.
```

Comment:
Could you be more clear here? I am not sure what this means

Reply:
boom.